src/App.tsx

```diff
@@ -3,12 +1,58 @@
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+  const [selectTodo, setSelectTodo] = useState<Todo | null>(null);
+  const [selectFilter, setSelectFilter] = useState('all');
+  const [hasLoading, setHasLoading] = useState(false);
+
+  useEffect(() => {
+    const loadTodos = async () => {
+      setHasLoading(true);
+      try {
+        const hasLoadTodos = await getTodos();
+
+        setTodos(hasLoadTodos);
+      } finally {
+        setHasLoading(false);
+      }
+    };
+
+    loadTodos();
+  }, []);
+
+  const isTodos = todos
```

Comment:
Consider renaming `isTodos` to `filteredTodos` or `displayedTodos` to better reflect its purpose as the list of todos that are filtered based on the current query and filter settings.